selfdrive/hardware/base.py

```diff
@@ -67,6 +71,12 @@ def get_sim_info(self):
   def get_network_strength(self, network_type):
     pass
 
+  def get_network_metered(self, network_type) -> bool:
+    if network_type in [NetworkType.none, NetworkType.wifi, NetworkType.ethernet]:
```

Comment:
```suggestion
    return network_type not in (NetworkType.none, NetworkType.wifi, NetworkType.ethernet)
```

Reply:
Ah, of course. There used to be other code here, but that's now gone.